provisioning/plugins/corim-nitro-decoder/test-harness/AWSNitroComid.json

```diff
@@ -0,0 +1,40 @@
+{
+    "lang": "en-GB",
+    "tag-identity": {
+      "id": "fb6789c3-147d-4e5f-8be0-4452b93c3767",
+      "version": 0
+    },
+    "entities": [
+      {
+        "name": "AWS",
+        "regid": "https://aws.amazon.com",
+        "roles": [
+          "tagCreator",
+          "creator",
+          "maintainer"
+        ]
+      }
+    ],
+    "triples": {
+      "attester-verification-keys": [
+        {
+          "environment": {
+            "class": {
+              "id": {
+                "type": "uuid",
+                "value": "51d1a77a-9908-49e0-92df-e9c872a9b0bd"
```

Comment:
what does this uuid represent?

Reply:
ok.  the environment identifier is used (indirectly) as a lookup key for the endorsements.  so it should be somewhat bound to the incoming evidence.